README.md

```diff
@@ -299,14 +310,16 @@ No local configuration is set
 
 More information about mbed CLI configuration is available in the [configuration section](#mbed-cli-configuration) of this document.
 
-#### Through mbed_settings.py
+#### Setting environment variable
 
-Edit `mbed_settings.py` to set your toolchain:
+For each of the compilers, there is a corresponding environment variable that is checked for the path to that compiler. The environment variables are as follows:
+* `MBED_ARM_PATH`: The path to the *base* directory of your ARM Compiler installation. This should be one directory up the directory containing the binaries for `armcc` and friends.
```

Comment:
Does this mean the directory above or before the directory containing the binaries for `armcc`, or does this mean something else?